Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Tooltips/ToolTip.cs

```diff
@@ -0,0 +1,540 @@
+        /// </summary>
+        public Transform ContentParentTransform
+        {
+            get
+            {
+                return contentParent.transform;
+            }
+        }
+
+        /// <summary>
+        /// is ToolTip active and displaying
+        /// </summary>
+        public bool IsOn
+        {
+            get
+            {
+                switch (masterTipState)
+                {
+                    case DisplayModeType.None:
+                    default:
+                        // Use our group state
+                        switch (groupTipState)
+                        {
+                            case DisplayModeType.None:
+                            default:
+                                // Use our local State
+                                switch (tipState)
+                                {
+                                    case DisplayModeType.None:
+                                    case DisplayModeType.Off:
+                                    default:
+                                        return false;
+
+                                    case DisplayModeType.On:
+                                        return true;
+
+                                    case DisplayModeType.OnFocus:
+                                        return HasFocus;
+                                }
+
+                            case DisplayModeType.On:
+                                return true;
+
+                            case DisplayModeType.Off:
+                                return false;
+
+                            case DisplayModeType.OnFocus:
+                                return HasFocus;
+                        }
+
+                    case DisplayModeType.On:
+                        return true;
+
+                    case DisplayModeType.Off:
+                        return false;
+
+                    case DisplayModeType.OnFocus:
+                        return HasFocus;
+                }
+            }
+        }
+
+        /// <summary>
+        /// does the ToolTip have focus.
+        /// </summary>
+        public bool HasFocus
+        {
+            get
+            {
+                return false;
+                //switch (ButtonState)
+                //{
+                //    case ButtonStateEnum.Targeted:
+                //    case ButtonStateEnum.ObservationTargeted:
+                //    case ButtonStateEnum.Pressed:
+                //        return true;
+
+                //    default:
+                //        return false;
+                //}
+            }
+        }
+
+        /// <summary>
+        /// Used for Content Change update in text
+        /// </summary>
+        public Action ContentChange;
+
+        /// <summary>
+        /// virtual functions
+        /// </summary>
+        protected virtual void OnEnable() {
+
+            //if (toolTipLine != null && toolTipLine.gameObject != gameObject)
```

Comment:
this looks to be outdated code, probably should be removed.

Reply:
Also addressed in my PR on his branch.